### ISSUE TITLE
Feat request: pubsubplus-connector-kafka-sink should packaging should be compatible with confluent-cli

### ISSUE THREAD

**Issue by fhussonnois** [SolaceProducts/pubsubplus-connector-kafka-sink#36]
Currently, the provided https://solaceproducts.github.io/pubsubplus-connector-kafka-sink/downloads/ is not compatible with the [confluent-hub client](https://docs.confluent.io/home/connect/confluent-hub/client.html).

```
 $ ./confluent-hub install pubsubplus-connector-kafka-sink-2.0.2.zip --component-dir /tmp

Unable to load component's manifest from a file 
/tmp/confluent-hub-tmp18383162373698779558/manifest.json
 
Error: Error while reading from/writing to filesystem
```

**Comment by ahmedtoumi** [comment]
Effectivement, il faut bidouiller le zid généré dans le dossier "target/distribution" en rajoutant le manifest.json manuellement.
A mon avis c'est un bug et pas une amélioration

**Comment by ahmedtoumi** [comment]
I created a PR to fix that issue https://github.com/SolaceProducts/pubsubplus-connector-kafka-sink/pull/72

**Comment by PhilippeKhalife** [comment]
@ahmedtoumi Thank for your contribution. The team will be having a look and responding to this soon.
